index.html

```diff
@@ -208,23 +212,64 @@ <h4 class="item_name">Frameworks</h4>
         </ul>
       </li>
       <li class="other_item">
-        <img class="item_logo" src="assets/skills.png" alt="language logo"/>
+        <img class="item_logo" src="assets/skills.png" alt="language logo" />
         <h4 class="item_name">Skills</h4>
         <ul class="items_sub_list">
           <li class="items_sub">Codekit</li>
           <li class="items_sub">GitHub</li>
           <li class="items_sub">Codepen</li>
-          
+
           <li class="items_sub">Gitlab</li>
           <li class="items_sub">Terminal</li>
         </ul>
       </li>
     </ul>
-    <section class="contact_me_section">
-      <div class="contact_text">
+    <footer class="portofolio_footer">
+      <div class="contact_me_section">
+        <h1 class="contact_form_title">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h1>
+        <form class="contact_form" action="https://formspree.io/f/mknegjnj" method="POST">
+          <input type="text" class="form_input" placeholder="Full Name" maxlength="30" required>
+          <input type="email" class="form_input" placeholder="Email address" required>
+          <textarea id="contact_text_area" class="form_input" placeholder="write your message here" maxlength="500"
```

Comment:
- [ ] An error occurs while trying to submit your form. FormSpree expects you to have the `name` attribute on all your form input elements. This is how FormSpree gets the data in your input. 
For instance:
```
<input type="email" name="email" class="form_input" placeholder="Email address" required>
```